README.md

```diff
@@ -93,6 +93,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
     * [**M-Droid**](https://github.com/SkyzohKey/M-Droid) <sup>**[[F-Droid](https://f-droid.org/app/com.mdroid)]**</sup>
     * [**Aurora Droid**](https://gitlab.com/AuroraOSS/auroradroid) <sup>**[[F-Droid](https://f-droid.org/app/com.aurora.adroid)]**</sup>
     * [**Foxy Droid**](https://github.com/kitsunyan/foxy-droid) <sup>**[[F-Droid](https://f-droid.org/app/nya.kitsunyan.foxydroid)]**</sup>
+    * [**IzzyOnDroid**](https://gitlab.com/sunilpaulmathew/izzyondroid) <sup>**[[F-Droid](https://f-droid.org/packages/in.sunilpaulmathew.izzyondroid)] | [[IzzyDroid](https://apt.izzysoft.de/fdroid/index/apk/in.sunilpaulmathew.izzyondroid)]**</sup>
```

Comment:
Please use the `/app/` url for F-Droid.

F-Droid is the only 3rd party link; izzy is not strictly FOSS.

Reply:
IzzyOnDroid is already in the App Store category.
It's not a F-Droid client app hence not in the F-Droid sub-category.

Reply:
Nice spot, thanks.

Reply:
> Nice spot, thanks.

So, should I add Izzy back?

Reply:
Izzy is included some lines below already.

Reply:
I'm going to fix this, so the PR is ready to merge :+1: